src/base/static/components/atoms/navigation.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import "./navigation.scss";
+
+const Link = ({ children, ...props }) => {
```

Comment:
Hmm, thinking about this atom a bit more, I wonder if a better name would be `ButtonLink`? I think we might want a more generic `Link` atom that is just a pair of `<a>` tags that render whatever `children` are passed into them.

But I also think this `ButtonLink` is useful, since it regularizes our button navigation styles.

Reply:
>I think we might want a more generic Link atom that is just a pair of <a> tags that render whatever children are passed into them.

Not sure I follow. Isn't that what this component is doing?

Reply:
Well the `Link` atom as-is will enforce button-like styling (i.e. the `variant` will be `flat` or `raised`). But on second thought I guess we'd just want to add a new variant like `unstyled`, perhaps as the default `variant` prop.